staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -33,7 +33,7 @@ const (
 
 var (
 	// Use buckets ranging from 25 ms to ~2.5 seconds.
```

Comment:
The current buckets generated by the expression underneath are: `[0.0025 0.00625 0.015625 0.0390625 0.09765625]` so the comment is actually still inaccurate.

Reply:
If we want to stay within the constraints of five buckets then my suggestion is `{0.0025, 0.015, 0.1, 1.0, 5.0}`. We should also update the comment to match the buckets.